fplus-lib/src/core/mod.rs

```diff
@@ -2521,6 +2549,40 @@ impl LDNApplication {
         }
     }
 
+    async fn check_and_handle_allowance(
+        db_multisig_address: &str,
+        new_allocation_amount: Option<String>,
+    ) -> Result<(), LDNError> {
+        let blockchain = BlockchainData::new();
+        match blockchain.get_allowance_for_address("f24siazyti3akorqyvi33rvlq3i73j23rwohdamuy").await {
```

Comment:
Leave this as it is just for testing purposes. We need to remove it when tagging into prod